src/ms3/bs4_parser.py

```diff
@@ -3785,18 +3906,12 @@ def set_instrument(self, staff_id: Union[str, int], trackname):
                 self.updated.update({staff_id: new_values["id"]})
             else:
                 # if there is no data for the trackname to update
-                fuzzy_matches = difflib.get_close_matches(
+                trackname_norm = difflib.get_close_matches(
```

Comment:
Here you have undone work by another collaborator and re-introduced a bug. Always review what you commit, please, @leobruneau.

Reply:
I'm really sorry about this and surprised since I don't at all recall writing such lines of code... In any case you are totally right, I should've checked before committing. I'm truly sorry for the inconvenience @johentsch